osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerRoomComposite.cs

```diff
@@ -31,6 +35,9 @@ protected override void LoadComplete()
         private void invokeUserJoined(MultiplayerRoomUser user) => Scheduler.AddOnce(UserJoined, user);
         private void invokeUserKicked(MultiplayerRoomUser user) => Scheduler.AddOnce(UserKicked, user);
         private void invokeUserLeft(MultiplayerRoomUser user) => Scheduler.AddOnce(UserLeft, user);
```

Comment:
Saving the fixes on these for a separate PR?

Reply:
Yup, I didn't find any immediate issue with these, but the playlist ones were causing issues and probably the source of some problems players experienced recently.